http/codegen/testdata/openapi_dsls.go

```diff
@@ -314,6 +314,24 @@ var ExtensionDSL = func() {
 	})
 }
 
+var SkipBodyEncodeDecodeDSL = func() {
+	Service("testService", func() {
+		Method("testEndpoint", func() {
+			Payload(Any, func() {
+				Example(`{"foo":"bar"}`)
+			})
+			Result(Any, func() {
+				Example(`{"foo":"bar"}`)
+			})
+			HTTP(func() {
+				POST("/")
+				SkipRequestBodyEncodeDecode()
+				SkipResponseBodyEncodeDecode()
+			})
+		})
+	})
+}
```

Comment:
Do you think we also need to test for the cases where the body has multiple attributes and some are used by other elements than the body (e.g. headers, params etc.)?